### PR TITLE
Empty input format prompt

### PR DESCRIPTION
**Pull request by CheahHaoYi**
Prompt user when they just press enter
Prompt format of Search Command if they are not followed correctly or are all empty

### CODE REVIEW

File: src/main/java/seedu/duke/command/SearchModuleCommand.java [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+import java.util.ArrayList;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 public class SearchModuleCommand extends Command {
     public static final String COMMAND_WORD = "search";
-    public static final String COMMAND_USAGE = "search [KEYWORD]";
-    public static final String COMMAND_DESCRIPTION = "List out all modules that contains KEYWORD"
-            + System.lineSeparator() + "\t * KEYWORD can either be module code or faculty.";
+    public static final String COMMAND_USAGE = "search (/code [MODULE_CODE] | /title [KEYWORD])";
```
Comment (kohnh) [inline src/main/java/seedu/duke/command/SearchModuleCommand.java:18]: should you include like /level or /sem

**Review by amitrahman1026 — APPROVED** [review]
LGTM

File: src/main/java/seedu/duke/command/SearchModuleCommand.java [hunk 2]
```diff
@@ -29,22 +30,25 @@ public class SearchModuleCommand extends Command {
...
 
-    public SearchModuleCommand(String input) {
+    public SearchModuleCommand(String input) throws YamomException {
         super(input.split("\\s"));
         params = Parser.parseParams(input);
+
+        toSearchModuleCode = params.getOrDefault("code", null);
+        toSearchModuleTitle = params.getOrDefault("title", null);
+        toSearchLevel = params.getOrDefault("level", null);
+        toSearchSemester = params.getOrDefault("sem", null);
```
Comment (kohnh) [inline src/main/java/seedu/duke/command/SearchModuleCommand.java:40]: are these variables saved? These variables are used in the execute function

**Review by kohnh — COMMENTED** [review]
Some changes may be required

**Comment by CheahHaoYi** [comment]
Fix #109

**Pull request merged**